RubrikPolaris/M365/Start-OperationalRecovery.ps1

```diff
@@ -94,6 +97,8 @@ function Start-OperationalRecovery() {
         [DateTime]$SharepointUntilTime,
         [Parameter(Mandatory=$False)]
         [Boolean]$ShouldSkipItemPermission,
+        [Parameter(Mandatory=$True)]
```

Comment:
Does this need to be mandatory if we have a false evaulation check @ L215?

Reply:
Yes, because this is an important option for recover. We want customers know what they are choosing for this option.